schemas/environmentsSchema.json

```diff
@@ -32,6 +32,15 @@
 					"sshLogin": {
 						"type": "string",
 						"description": "The login used to ssh into the Profile Server."
+					},
+					"serverType": {
+						"type": "string",
+						"description": "The server type. Default value is \"SCA$IBS\"."
+					},
+					"encoding": {
+						"type": "string",
+						"description": "Encoding to use in request/response. The default value is \"utf8\".",
```

Comment:
Or simply use this setting as Target (Host side encoding) and then VSCode users can use whatever they like and it will be converted to this encoding.